scripts/jenkins-yoctobuild-build.sh

```diff
@@ -862,8 +863,10 @@ publish_artifacts() {
         local client_version=$($WORKSPACE/integration/extra/release_tool.py --version-of mender --in-integration-version HEAD)
         local mender_artifact_version=$($WORKSPACE/integration/extra/release_tool.py --version-of mender-artifact --in-integration-version HEAD)
 
-        s3cmd --cf-invalidate -F put $WORKSPACE/go/bin/mender-artifact s3://mender/mender-artifact/${mender_artifact_version}/
-        s3cmd setacl s3://mender/mender-artifact/${mender_artifact_version}/mender-artifact --acl-public
+        for bin in mender-artifact-darwin mender-artifact-linux mender-artifact-windows.exe; do
+            s3cmd --cf-invalidate -F put $WORKSPACE/go/src/github.com/mendersoftware/mender-artifact/$bin s3://mender/mender-artifact/${mender_artifact_version}/
+            s3cmd setacl s3://mender/mender-artifact/${mender_artifact_version}/$bin --acl-public
+        done
```

Comment:
Do you think we should also upload the original, no-suffix linux version? Or should we stick to these new variants? If we do, then we need to change the download links in mender-docs.

Reply:
I thought about it before.

We need to modify the docs anyway to add the Mac version, so I think it will look more consistent to keep the platform suffix on the three binaries. Conceptually, otherwise assumes Linux is the default.

However, I had a concern on that someone might have automated the download of the `mender-artifact` tool and in the next release such automation will fail. I assessed this to be a low impact error and decided to cope with it.

What do you think?

Reply:
Yes, I think I agree. Otherwise there will always be potential confusion about which platform the suffixless version belongs to. I also think it is a very low impact change, and it is anyway in a versioned directory, so it won't break any existing downloads. Only when they update their version number.